packetbeat/docs/fields.asciidoc

```diff
@@ -6,7 +6,7 @@ This file is generated! See etc/fields.yml and scripts/generate_field_docs.py
 [[exported-fields]]
 == Exported Fields
 
-This document describes the fields that are exported by Docs/Fields.Asciidoc. They are
+This document describes the fields that are exported by Packetbeat. They are
```

Comment:
https://github.com/elastic/beats/pull/1107#discussion_r55415324
Fixed in https://github.com/elastic/beats/pull/1129